tests/e2e/cucumber/features/integrations/search.feature

```diff
@@ -0,0 +1,91 @@
+Feature: link
+  As a user
+  I want to search for resources
+  So that I can find them quickly
+
+  Background:
+    Given "Admin" sets the default folder for received shares to "Shares"
+    And "Admin" disables share auto accepting
+
+  Scenario: Search in personal spaces
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+      | Brian |
+    When "Brian" logs in
+    And "Brian" opens the "files" app
+    And "Brian" creates the following resources
+      | resource          | type   |
+      | folder_from_brian | folder |
+    And "Brian" uploads the following resources
+      | resource          |
+      | new-lorem-big.txt |
+    And "Brian" shares the following resource using the sidebar panel
+      | resource          | recipient | type | role   |
+      | folder_from_brian | Alice     | user | viewer |
+      | new-lorem-big.txt | Alice     | user | viewer |
+    And "Brian" logs out
+    When "Alice" logs in
+    And "Alice" navigates to the shared with me page
+    And "Alice" accepts the following share
+      | name              |
+      | folder_from_brian |
+      | new-lorem-big.txt |
+    And "Alice" opens the "files" app
+    And "Alice" creates the following resources
+      | resource                   | type   |
+      | folder                     | folder |
+      | FolDer                     | folder |
+      | PARENT/child               | folder |
+      | PARENT/child-one           | folder |
+      | PARENT/child-one/child-two | folder |
+    And "Alice" enables the option to display the hidden file
+    And "Alice" uploads the following resources
+      | resource         |
+      | new-lorem.txt    |
+      | .hidden-file.txt |
+    And "Alice" searches "new" using the global search
+    Then following resources should be displayed in the search list for user "Alice"
+      | resource          |
+      | new-lorem.txt     |
+      | new-lorem-big.txt |
```

Comment:
comment out this line until  https://github.com/owncloud/web/issues/7629 will be fixed. After that we can merge your PR

Reply:
Done in latest commit